tests/test_utils.py

```diff
@@ -250,6 +250,42 @@ def test_tracks_to_napari_ndim_inference(ndim: int):
     assert data.shape[-1] == ndim + 2
 
 
+def test_napari_to_tracks(sample_tracks):
+    """Test that a napari Tracks layer can be converted to a list of Tracklets.
+
+    First convert tracks to a napari layer, then convert back and compare.
+    """
+
+    data, properties, graph = utils.tracks_to_napari(sample_tracks)
+    tracks = utils.napari_to_tracks(data, properties, graph)
+
+    properties_to_compare = [
+        "ID",
+        "t",
+        "x",
+        "y",
+        # "z",  # z-coordinates are different
```

Comment:
I think this makes sense for this test.

Reply:
I think it's the same issue with dummy objects having non-zero z coordinates even in 2D datasets, but these are lost when converting to napari